server/pulp/server/managers/repo/sync.py

```diff
@@ -79,8 +79,7 @@ def sync(repo_id, sync_config_override=None):
         repo_importer = importer_manager.get_importer(repo_id)
 
         # Assemble the data needed for the sync
-        conduit = RepoSyncConduit(repo_id, repo_importer['id'], RepoContentUnit.OWNER_TYPE_IMPORTER,
-                                  repo_importer['id'])
+        conduit = RepoSyncConduit(repo_id, repo_importer['id'])
 
         call_config = PluginCallConfiguration(importer_config, repo_importer['config'],
                                               sync_config_override)
```

Comment:
marking where I left off for lunch :smile:  will continue when I get back

:hamburger: